Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/MeshComponentController.cpp

```diff
@@ -699,6 +701,81 @@ namespace AZ
             return Aabb::CreateNull();
         }
 
+        void MeshComponentController::GetVisibleGeometry(
+            [[maybe_unused]] const AZ::Aabb& bounds, AzFramework::VisibleGeometryContainer& geometryContainer) const
+        {
+            // Attempt to copy the triangle list geometry data out of the model asset into the visible geometry structure
+            const auto& modelAsset = GetModelAsset();
+            if (!modelAsset.IsReady() || modelAsset->GetLodAssets().empty())
+            {
+                AZ_Warning("MeshComponentController", false, "Unable to get geometry because mesh asset is not ready or empty.");
+                return;
+            }
+
+            // This will only extract data from the first LOD. It might be necessary to make the LOD selectable.
+            const auto& lodAsset = modelAsset->GetLodAssets().front();
+            if (!lodAsset.IsReady())
+            {
+                AZ_Warning("MeshComponentController", false, "Unable to get geometry because selected LOD asset is not ready.");
+                return;
+            }
+
+            const AZ::Name positionName{ "POSITION" };
```

Comment:
MeshFeatureProcessor has **static const char* PositionSemantic = "POSITION";**. Wonder if we could move this to a central location which can be used by everyone.